tango_simlib/compat.py

```diff
@@ -17,7 +17,18 @@
 if future.utils.PY2:
 
     def ensure_native_ascii_str(value):
-        """Coerce unicode string or bytes to native string type (ascii encoding)."""
+        """Coerce unicode string or bytes to native string type (ascii encoding).
+
+        Parameters
+        ----------
+        value: String
```

Comment:
Suggest using the Python types rather the "String", to be more specific.
`str or unicode`